assets/svg/icons/VolunteerActivism.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" enable-background="new 0 0 26 26" viewBox="-2 -1 28 28"><g><rect fill="none" height="24" width="24"/></g><g><g><rect height="11" width="4" x="1" y="11"/><path d="M16,3.25C16.65,2.49,17.66,2,18.7,2C20.55,2,22,3.45,22,5.3c0,2.27-2.91,4.9-6,7.7c-3.09-2.81-6-5.44-6-7.7 C10,3.45,11.45,2,13.3,2C14.34,2,15.35,2.49,16,3.25z"/><polygon points="22,17 13,17 10.91,16.27 11.24,15.32 13,16 17,16 17,14 8.97,11 7,11 7,20.02 14,22 22,19"/></g></g></svg>
```

Comment:
I agree, it is an awkward icon. The heart icon is also a tough one since it is used widely as a favoriting icon, and not a giving link, at least in media related UI's.

Reply:
+1. Its a hard one.